src/test/spec/json/retryable-reads/retryable-reads.rst

```diff
@@ -0,0 +1,667 @@
+===============
```

Comment:
I don't think we need to check in the entire retryable-reads spec into the repository, just the test files should be sufficient.

Reply:
In a similar vein, is it necessary to include the `.yml` files alongside the `.json` since the runner doesn't use them?

Reply:
I think we keep the .yml files around for the comments in them, though you're right that we don't really need them.

Reply:
I personally like having the yaml files during development/while I'm debugging failed tests, as they're much easier for me to read.